src/storage/index.ts

```diff
@@ -648,14 +683,15 @@ export async function putFile(
 export async function deleteFile(
   path: string, 
   options?: {
-    wasSigned?: boolean;
+    wasSigned?: boolean,
+    gaiaHubConfig?: GaiaHubConfig
   },
   caller?: UserSession
 ) {
   if (!caller) {
     caller = new UserSession()
   }
-  const gaiaHubConfig = await caller.getOrSetLocalGaiaHubConnection()
+  const gaiaHubConfig = options.gaiaHubConfig || await caller.getOrSetLocalGaiaHubConnection()
```

Comment:
`options` can be undefined and cause this line to throw. Can be reproduced with `userSession.deleteFile('anything')`